main.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+#include "./DrawingEngineFactory/DrawingEngineFactory.h"
+#include "./Logs/Logs.h"
+#include <memory>
+#include <vector>
+
+using namespace std;
+
+int main(){
+	writeToLogs("App start");
+	//creating drawing engine factory with specific style of drawing
+	DrawingEngineFactory def = DrawingEngineFactory(0);
+	//creating drawing engine and taking poiter to it
+	shared_ptr<DrawingEngine> engine = def.createEngine(3,4, 'r', 'w');
```

Comment:
the way you defined drawing engine constructor will not make sense for FTXUI, also it will be hard to extend for borders or any other thing you would might like to add